core/queryparser/sparql/src/main/java/org/eclipse/rdf4j/query/parser/sparql/PrefixDeclProcessor.java

```diff
@@ -55,6 +55,21 @@ public class PrefixDeclProcessor {
 	 * @throws MalformedQueryException If the query contains redefined prefixes or qnames that use undefined prefixes.
 	 */
 	public static Map<String, String> process(ASTOperationContainer qc) throws MalformedQueryException {
+		return process(qc, null);
```

Comment:
```suggestion
		return process(qc, Collections.emptyMap());
```
Passing an empty map here would make the overloaded method simpler.